README.md

```diff
@@ -96,11 +99,16 @@ SparseML enables you to create a sparse model trained on your dataset in two way
 - [Sparse Transfer Learning with the CLI](integrations/ultralytics-yolov5/tutorials/sparse-transfer-learning.md)
```

Comment:
```suggestion
- [Sparse Transfer Learning With the CLI](integrations/ultralytics-yolov5/tutorials/sparse-transfer-learning.md)
```

Reply:
APA style guide is used usually w/ tech pubs. "With" is always capped in titles - heads up.